From 83d20cd09143d4684176a0de90f9acc988d394d6 Mon Sep 17 00:00:00 2001
From: Kirill Kapranov <kirill.kapranov@compulab.co.il>
Date: Tue, 16 Jun 2020 15:45:22 +0300
Subject: [PATCH 039/108] Fix storages numeration order

Make u-boot enumerate storages in the same order as previous versions (2018.03)

Signed-off-by: Kirill Kapranov <kirill.kapranov@compulab.co.il>
---
 arch/arm/dts/ucm-imx8m-mini.dts | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm/dts/ucm-imx8m-mini.dts b/arch/arm/dts/ucm-imx8m-mini.dts
index 3d8be3a09a..b191e87755 100644
--- a/arch/arm/dts/ucm-imx8m-mini.dts
+++ b/arch/arm/dts/ucm-imx8m-mini.dts
@@ -16,6 +16,7 @@
 
 #include "fsl-imx8mm.dtsi"
 
+
 / {
 	model = "CompuLab UCM-iMX8M-Mini board";
 	compatible = "cpl,ucm-imx8m-mini", "fsl,imx8mm-evk", "fsl,imx8mm";
@@ -25,6 +26,12 @@
 		stdout-patch = &uart3;
 	};
 
+	aliases {
+		mmc0 = &usdhc2;
+		mmc1 = &usdhc3;
+		/delete-property/ mmc2;
+	};
+
 	reg_usdhc2_vmmc: regulator-usdhc2 {
 		compatible = "regulator-fixed";
 		regulator-name = "VSD_3V3";
-- 
2.11.0

